src/pages/RestrictedAction/Workspace/WorkspaceAdminRestrictedAction.tsx

```diff
@@ -0,0 +1,72 @@
+import React, {useCallback} from 'react';
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import Button from '@components/Button';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import * as Illustrations from '@components/Icon/Illustrations';
+import ImageSVG from '@components/ImageSVG';
+import ScreenWrapper from '@components/ScreenWrapper';
+import ScrollView from '@components/ScrollView';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as Report from '@libs/actions/Report';
+import Navigation from '@libs/Navigation/Navigation';
+import * as PolicyUtils from '@libs/PolicyUtils';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+
+type WorkspaceAdminRestrictedActionProps = {
+    policyID: string;
+};
+
+function WorkspaceAdminRestrictedAction({policyID}: WorkspaceAdminRestrictedActionProps) {
+    const {translate} = useLocalize();
+    const [policy] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`);
+    const styles = useThemeStyles();
+
+    const openAdminsReport = useCallback(() => {
+        const reportID = `${PolicyUtils.getPolicy(policyID)?.chatReportIDAdmins}` ?? '-1';
+        Report.openReport(reportID);
+        Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(reportID));
+    }, [policyID]);
+
+    return (
+        <ScreenWrapper
+            includeSafeAreaPaddingBottom
+            testID={WorkspaceAdminRestrictedAction.displayName}
+        >
+            <HeaderWithBackButton
+                title={translate('workspace.restrictedAction.restricted')}
+                onBackButtonPress={Navigation.goBack}
+            />
+            <ScrollView
+                style={[styles.p5, styles.pt0]}
+                contentContainerStyle={styles.flexGrow1}
+            >
+                <View style={[styles.flex1, styles.alignItemsCenter, styles.justifyContentCenter, styles.mb15]}>
+                    <ImageSVG
+                        src={Illustrations.LockClosedOrange}
+                        width={136}
```

Comment:
why 136x136? this should be our standard simple illustration height which is 48x48, and you can use the var we have for that as well.